juniper/src/integrations/chrono_tz.rs

```diff
@@ -0,0 +1,66 @@
+/*!
+
+# Supported types
+
+| Rust Type               | JSON Serialization     | Notes                                     |
+|-------------------------|------------------------|-------------------------------------------|
+| `Tz`                    | Tz database name       |                                           |
+
+*/
+use chrono_tz::Tz;
+
+use crate::{
+    parser::{ParseError, ScalarToken, Token},
+    value::ParseScalarResult,
+    Value,
+};
+
+graphql_scalar!(Tz as "Tz" where Scalar = <S>{
```

Comment:
Nit: Add a space after `<S>`

Reply:
thanks for noticing this. Wonder why `rustfmt` didn't take care of this 🤔